\textcolor{UNIR}{\fontspec{Calibri Light}\fontsize{18}{28}\selectfont
Abstract}

Structural variants (SVs) represent genomic alterations that encompass deletions, 
insertions, and rearrangements of segments ranging from kilobases to entire 
chromosomes. Despite their significance as biomarkers in oncological diseases, 
these variants have remained relatively understudied compared to single nucleotide 
variants, largely due to the inherent limitations of short-read sequencing 
technologies that have dominated large-scale genome sequencing projects. This 
landscape has undergone a transformative shift with the emergence of long-read 
sequencing technologies, which have enabled the achievement of the first truly 
complete human reference genome, from telomere to telomere, successfully resolving 
gaps that short reads could not address. This project focuses on conducting a 
comprehensive evaluation of the performance of long-read-based structural variant 
callers, specifically within the context of tumor evolution analysis. To address 
the limited availability of suitable datasets, we have developed specialized 
workflows that leverage high-performance computing resources to generate synthetic 
data with customized SVs, thereby facilitating robust benchmarking of various 
structural variant detection methods. This computational approach enables the 
systematic assessment of SV detection algorithms under controlled conditions, 
providing valuable insights into their performance and reliability.

\vspace{0.5cm}

\noindent\textbf{Keywords:} 
    
Structural Variants, Cancer Genomics, Variant Calling, 
Nanopore Sequencing, Long-reads.

